crates/ruff_python_formatter/tests/snapshots/format@statement__import_from.py.snap

```diff
@@ -40,6 +56,54 @@ from a import (
     aksjdhflsakhdflkjsadlfajkslhfdkjsaldajlahflashdfljahlfksajlhfajfjfsaahflakjslhdfkjalhdskjfa as sdkjflsdjlahlfd,
 )
 from aksjdhflsakhdflkjsadlfajkslhfdkjsaldajlahflashdfljahlfksajlhfajfjfsaahflakjslhdfkjalhdskjfa import *
+
+
+from a import bar  # comment
+
+from a import (  # comment
+    bar,
+    bar,
+    bar,
+    bar,
+    bar,
+    bar,
+    bar,
+    bar,
+    bar,
+    bar,
+    bar,
+    bar,
+    bar,
+    bar,
+    bar,
+    bar,
+)
```

Comment:
For me, the current style (same as black's style) matches my expectations better:
```python
from a import b, dkasjhfaslhdflkjashdlfjkashldfkjahlsdkjfha, sdkfjhlasdfkjlhasdfkjhasdflkjhasd # c
from a import b, dkasjhfaslhdflkjashdlfjkashldfkjahlsdkjfha, sdkfjhlasdfkjlhasdfkjhasdflkjhasd # c

def f():
    pass
```
into
```python
from a import (
    b,
    dkasjhfaslhdflkjashdlfjkashldfkjahlsdkjfha,
    sdkfjhlasdfkjlhasdfkjhasdflkjhasd,
)  # c
from a import (
    b,
    dkasjhfaslhdflkjashdlfjkashldfkjahlsdkjfha,
    sdkfjhlasdfkjlhasdfkjhasdflkjhasd,
)  # c


def f():
    pass
```

Reply:
Candidly I think what Black is doing is wrong (the comment is on the import, not the member).

Reply:
Candidly I think what Black is doing is wrong (the comment is on the import, not the member).

Reply:
For example, given:

```python
from foo import (  # comment
  bar,
  baz
)
```

Black will then format as:

```python
from foo import bar, baz  # comment
```

Under Black's logic, if that import then breaks again by getting too long, it will move the comment:

```python
from foo import (
  bar,
  baz,
)  # comment
```

Reply:
I'm actually inclined to keep what we have here, but I'm definitely open to disagreement. What do you think?

Reply:
Eh, I will just revert it for now for simplicity.